src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/fieldFilterTypes/SelectionFieldFilterType.js

```diff
@@ -44,11 +43,15 @@ class SelectionFieldFilterType extends AbstractFieldFilterType<?Array<string | n
         });
 
         this.valueDisposer = autorun(() => {
-            const {value = []} = this;
+            const value = toJS(this.value || []);
 
-            if (!equals(toJS(value), untracked(() => this.selectionStore.ids))) {
+            if (!equals(value, untracked(() => this.selectionStore.ids))) {
                 this.selectionStore.loadItems(value);
             }
+
+            if (!equals(value, untracked(() => this.selectValue))) {
```

Comment:
this is not tested. but i am not sure if there was a test for the previous implementation